pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -582,20 +582,39 @@ private void createDerivedColumnV1Indices(String column, FunctionEvaluator funct
       int numDocs = _segmentMetadata.getTotalDocs();
       Object[] outputValues = new Object[numDocs];
       PinotDataType outputValueType = null;
+      int loggedErrorCount = 0;
+      long lastLoggedTime = System.currentTimeMillis();
+
+      FieldSpec fieldSpec = _schema.getFieldSpecFor(column);
+      Object defaultNullValue = fieldSpec.getDefaultNullValue();
       for (int i = 0; i < numDocs; i++) {
         for (int j = 0; j < numArguments; j++) {
           inputValues[j] = valueReaders.get(j).getValue(i);
         }
-        Object outputValue = functionEvaluator.evaluate(inputValues);
-        outputValues[i] = outputValue;
+        try {
+          Object outputValue = functionEvaluator.evaluate(inputValues);
+          outputValues[i] = outputValue;
+        } catch (Exception e) {
+          if (!errorOnFailure) {
+            // Try to not flush the logger, sample every 1000 errors or 10 seconds
```

Comment:
We can consider only log one line after processing all the values for the column

Reply:
done.